testing/tpm2-tss/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Alexander Sack <asac@pantacor.com>
+# Maintainer: Alexander Sack <asac@pantacor.com>
+pkgname="tpm2-tss"
+pkgver="2.3.1"
+pkgrel=0
+pkgdesc="TPM2 tss library"
+url="https://github.com/tpm2-software/tpm2-tss/"
+arch="all"
+license="BSD-2-Clause"
+makedepends="openssl-dev doxygen linux-headers cmocka-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://github.com/tpm2-software/tpm2-tss/releases/download/${pkgver}/tpm2-tss-${pkgver}.tar.gz"
+
```

Comment:
You forgot to also add `options="!check"` here

Reply:
hmm. true. i added it to tools only. sorry. fixed.

Reply:
No worries :)

Will push once home, thank you for your work